src/MarkersHandler.js

```diff
@@ -325,7 +325,7 @@ var markersHandler = function (vjs, adMarkerStyle) {
 				var nextMarkerTime;
 
 				// post-roll support
-				if (currentTime === player.duration()) {
+				if (Math.abs(player.duration() - currentTime) < 0.1) {
```

Comment:
I think .1 is a little too tight - I'd go .25, which give you a .5 margin for error overall (.25 shorter or longer than expected).